script/bootstrap

```diff
@@ -18,9 +23,23 @@ fi
 ruby_version=`cat .ruby-version`
 
 echo "==> Installing Ruby: $ruby_version"
-ruby-install ruby $ruby_version --no-reinstall
-source /usr/local/opt/chruby/share/chruby/chruby.sh
-chruby $ruby_version
+if command_exists rbenv
+then
+    echo 'rbenv found'
+    rbenv install -s $ruby_version
+elif command_exists ruby-install
+then
+    echo 'ruby-install found'
+    ruby-install ruby $ruby_version --no-reinstall
+    source /usr/local/opt/chruby/share/chruby/chruby.sh
+    chruby $ruby_version
+else
+    echo 'Did not find rbenv or ruby-install!'
```

Comment:
I just wrote:

> This is fine, but it should never happen, because `ruby-install` and `chruby` are in the `Brewfile`.

Then I double checked that file and those things aren't there. I don't remember them ever not being there, but they never were. 

I think we should add both of them to `Brewfile` so that this script Just Works™. 

What do you think?

Reply:
i dont want to surprise install another ruby manager if people already do ruby development and come to this project.

It seems like a lot of people would already have `rbenv` or `rvm` installed. I had not heard of `ruby-install` until I ran into an error with `script/update`

Reply:
The let’s add rbenv to Brewfile instead. I’d like these scripts to work as standalone as possible.